api/live_testing/main.py

```diff
@@ -0,0 +1,157 @@
+#!/bin/env python3
+
+import requests
+import click
+import json
+
+routes = [
```

Comment:
Is there any particular reason these routes were chosen to be tested? Can/should they be specified externally to the tool as well?

Reply:
I chose them on the basis of trying to provide coverage of as much of our API as possible. I could probably have added a few more, but got bored :see_no_evil: 

I was thinking of providing them in a seperate json / yaml or something. Maybe worth it for a clean separating of code / configuration, but would require overhead of some parsing code.

Reply:
@jamieparkinson now specified in `routes.json`